lib/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_mlir_library(TTMLIR STATIC RegisterAll.cpp
     MLIRTTDialect
     MLIRTTIRDialect
     MLIRTTIRTransforms
+    MLIRTTConversions
```

Comment:
I named this lib to be in accordance with current names, but to me it makes more sense to call them TTMLIR<LIB_NAME>

Reply:
I agree, we should keep changing our libs to use `TTMLIR` prefix to differentiate from default `MLIR` ones. This should be `TTMLIRConversions`.

However, `MLIRTTIRTransforms` for example, should be renamed, in another PR, to `TTMLIRTTIRTransforms`, and NOT `TTMLIRIRTransforms`.

Reply:
Tracked by https://github.com/tenstorrent/tt-mlir/issues/232.